backend/app/controllers/spree/admin/promotions_controller.rb

```diff
@@ -28,7 +26,7 @@ def create
 
       def load_bulk_code_information
         @promotion_builder = Spree::PromotionBuilder.new(
-          base_code: @promotion.codes.first.try!(:value),
+          base_code: @promotion.base_code || @promotion.codes.first.try!(:value),
```

Comment:
They aren't exactly interchangeable, but there's no good way to write a migration to set that data, because that data simply isn't there -- that's what this PR fixes. 

I suppose you could write a migration that looked through all existing promotions, _if_ they have promotion codes and _if_ those codes all begin with the same prefix (assuming that prefix is always followed by an underscore, which the existing CodeBuilder does, but you could hypothetically have a custom CodeBuilder which does something else), then extract that prefix and set it as base_code?

That seemed far too error-prone and weird to try, to me.